runtime/lua/vim/health.lua

```diff
@@ -420,10 +425,15 @@ function M._check(mods, plugin_names)
       M.error('The healthcheck report for "' .. name .. '" plugin is empty.')
     end
 
+    local report = ('%2d ✅ %2d ⚠️ %2d ❌'):format(
```

Comment:
probably shouldn't show warn/error icon unless they are >0

Reply:
and I don't think we need to show the number of of OKs in the header; those (numbers) are not very useful.